docs/resources/fgs_function.md

```diff
@@ -193,7 +232,13 @@ The following arguments are supported:
 
 * `network_id` - (Optional, String) Specifies the network ID of subnet.
 
--> **NOTE:** An agency with VPC management permissions must be specified for the function.
+  -> An agency with VPC management permissions must be specified for the function.
+
+* `dns_list` - (Optional, String) Specifies the private DNS configuration of the function network.
+  private DNS list are associated to the function by a string in the following format:  
```

Comment:
The first letter of a sentence needs to be capitalized.

Reply:
done.